profiles/minhaj-313.json

```
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/107562768/226933817-9212dc22-f422-44d8-8aa2-888bb6d43878.png",
+  "issueId": 1766,
+  "name": "Shaikh Minhaj",
+  "username": "minhaj-313"
+}
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```